jvb/src/main/kotlin/org/jitsi/videobridge/message/BridgeChannelMessage.kt

```diff
@@ -338,6 +377,12 @@ class ForwardedEndpointsMessage(
  * TODO: consider and adjust the format of videoConstraints. Do we need all of the VideoConstraints fields? Document.
  */
 class SenderVideoConstraintsMessage(val videoConstraints: VideoConstraints) : BridgeChannelMessage(TYPE) {
+    /**
+     * Serialize manually because it's faster than either Jackson or json-simple.
+     * Note that we depend on `VideoConstraints.toString` producing JSON.
```

Comment:
I actually meant to put the comment in `VideoConstraints`, not here -- but if you say breaking that will cause the tests to fail, that's fine.